modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CP2SubscriptionManagement.testcase

```diff
@@ -23,109 +24,155 @@ definition {
 
-		CustomerPortal.enterOnProject(
-			status = "Future",
-			title = "Test Account 05");
+			CustomerPortal.enterOnProject(
+				status = "Future",
+				title = "Test Account Portal");
+		}
 
-		AssertElementNotPresent(locator1 = "CustomerPortalSite#SUBSCRIPTION_NAVBAR1");
+		task ("Assert that the tab is being displayed") {
+			AssertElementNotPresent(locator1 = "CustomerPortalSite#SUBSCRIPTION_NAVBAR1");
+		}
 	}
 
-	@description = "LPS-147034 - Verify if when the project has up to 4 options in the tabs for subscription management, the tabs are listed side by side"
-	@ignore = "true"
+	@description = "Automation ID: LPS-147034 | Test Summary: Verify if when the project has up to 3 options in the tabs for subscription management, the tabs are listed side by side"
 	@priority = "4"
 	test LessThan5TabsAreSideBySide {
-		SetWindowSize(value1 = "1920,1080");
+		task ("Width desktop") {
+			SetWindowSize(value1 = "1920,1080");
+		}
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.enterOnProject(title = "Test Account 06");
+			CustomerPortal.enterOnProject(title = "Test Account Commerce");
+		}
 
-		CustomerPortal.viewSubscriptionManagementTab(productTab = "DXP Cloud,Commerce,Enterprise Search");
+		task ("Assert that the signatures are being displayed in the tab") {
+			CustomerPortal.viewSubscriptionManagementTab(productTab = "LXC-SM,Commerce,Enterprise Search");
+		}
 	}
 
-	@description = "LPS-147022 - Verify if when the user doesn't have a subscription then a tab will not be shown"
+	@description = "Automation ID: LPS-147022 | Test Summary: Verify if when the user doesn't have a subscription then a tab will not be shown"
 	@priority = "4"
 	test TabDontAppearWithoutSubscription {
-		SetWindowSize(value1 = "1920,1080");
+		task ("Width desktop") {
+			SetWindowSize(value1 = "1920,1080");
+		}
 
-		ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
+		task ("Go to CP2.0 and join a project") {
+			ApplicationsMenu.gotoSite(key_site = "Customer Portal Site");
 
-		CustomerPortal.enterOnProject(title = "Test Account 01");
+			CustomerPortal.enterOnProject(title = "Test Account Partnership");
+		}
 
-		AssertElementPresent(locator1 = "CustomerPortalSite#SUBSCRIPTION_NAVBAR1");
+		task ("Assert that no tabs are displayed") {
+			AssertElementNotPresent(locator1 = "CustomerPortalSite#SUBSCRIPTION_NAVBAR");
+		}
 	}
 
-	@description = "LPS-147035 - Verify if when a project has at least 5 options in the tabs for subscription management, the tab list turns into a dropdown list"
+	@description = "Automation ID: LPS-147035 | Test Summary: Verify if when a project has at least 5 options in the tabs for subscription management, the tab list turns into a dropdown list"
 	@priority = "4"
 	test Tabs5OrMoreBecamesDropdownList {
-		SetWindowSize(value1 = "1920,1080");
+		task ("Width desktop") {
```

Comment:
maybe it should be "Given I set up the width of window desktop"

Reply:
Done.